pkgs/servers/home-assistant/component-packages.nix

```diff
@@ -1035,7 +1035,8 @@
       ]; # missing inputs: decora
     "decora_wifi" =
       ps: with ps; [
-      ]; # missing inputs: decora-wifi
+      decora-wifi
```

Comment:
Can you split this change off into a separate commit?

Reply:
And use the script i mentioned to generate the change in the first place.